Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/EditorMeshComponent.cpp

```diff
@@ -253,6 +254,12 @@ namespace AZ
             AzToolsFramework::ToolsApplicationEvents::Bus::Broadcast(
                 &AzToolsFramework::ToolsApplicationEvents::InvalidatePropertyDisplay,
                 AzToolsFramework::Refresh_EntireTree);
+
+            // Register this component with the editor mode feedback system
```

Comment:
should that come before the refresh tree?

Reply:
Fixed 👍